### PR TITLE
Updated to Klaros 4.12.7.

### PR DESCRIPTION
**Pull request by stolp**

### CODE REVIEW

File: MariaDB/db/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM verit/klaros-mariadb_db:4.12.6
+FROM verit/klaros-mariadb_db:4.12.7
```
Comment (BreadSpoon) [inline MariaDB/db/Dockerfile:1]: The tag "4.12.7" has not yet been released in Docker Hub.
That also counts for all other images.

**Comment by BreadSpoon** [comment]
It looks like there is an issue with the workflow. Working on it #37

**Comment by stolp** [comment]
That may explain my inability to resolve this, thanks!

**Comment by BreadSpoon** [comment]
Can you try to update your branch?

**Comment by stolp** [comment]
I am unsure on how to pull the changes

**Pull request merged**